x-pack/platform/plugins/shared/fleet/server/types/rest_spec/fleet_server_policy_config.ts

```diff
@@ -39,6 +45,22 @@ export const PutFleetServerHostRequestSchema = {
     is_default: schema.maybe(schema.boolean({ defaultValue: false })),
     is_internal: schema.maybe(schema.boolean()),
     proxy_id: schema.nullable(schema.string()),
+    certificate_authorities: schema.maybe(schema.oneOf([schema.literal(null), schema.string()])),
+    certificate: schema.maybe(
+      schema.nullable(schema.oneOf([schema.literal(null), schema.string()]))
```

Comment:
`schema.nullable` is not needed here

```suggestion
      schema.oneOf([schema.literal(null), schema.string()])
```